docker/Dockerfile

```diff
@@ -277,7 +277,7 @@ RUN --mount=type=cache,target=${CCACHE_DIR} \
   --mount=type=bind,from=rosdep-universe-sensing-perception-depend,source=/autoware/src/universe/autoware.universe/sensing/autoware_cuda_utils,target=/autoware/src/universe/autoware.universe/sensing/autoware_cuda_utils \
   source /opt/ros/"$ROS_DISTRO"/setup.bash \
   && source /opt/autoware/setup.bash \
-  && /autoware/build_and_clean.sh ${CCACHE_DIR} /opt/autoware
+  && /autoware/build_and_clean.sh ${CCACHE_DIR} /opt/autoware true
```

Comment:
https://github.com/autowarefoundation/autoware/blob/35911067370ca0130608f1fec3eafaab2a4ce8e6/docker/scripts/build_and_clean.sh

the current script doesn't make use of the `true` bool added here.

Is there a reason this argument was included in this PR?

Reply:
Ah sorry, it is being added below :sweat_smile: my bad :D